packages/esm-patient-common-lib/src/useVitalsConceptMetadata.tsx

```diff
@@ -17,11 +17,22 @@ export function useVitalsConceptMetadata() {
   const conceptUnits = conceptMetadata?.length
     ? new Map<string, string>(conceptMetadata.map((concept) => [concept.uuid, concept.units]))
     : new Map<string, string>([]);
+
+  const conceptRanges = conceptMetadata?.length
+    ? new Map<string, { lowAbsolute: number | string | null; highAbsolute: number | string | null }>(
```

Comment:
Do we need to include the "string" type here? These should basically always be numbers. Also, note that the values may be unset (i.e., undefined). If `undefined` isn't a valid value for `min` or `max` we should probably ensure that the value will be `null`, e.g., `{ lowAbsolute: concept.lowAbsolute ?? null }`.